presentations/en/GCcode-version-control-system-esdc-gc.md

```diff
@@ -0,0 +1,175 @@
+
+Make all non-sensitive data, information, and new code developed in delivery of services open to the outside world for sharing and reuse under an open licence
+
+---
+
+## Current State
+
+Each department manages its own internal VCS/Project management platforms
+
+> ESDC has TFS, SVN and GitLab (Azure)
+
+For open source code and working in the open, SaaS are recomended
+
+> ESDC is using mostly Github.com
+
+--
+
+### GCcode
+
+SSC maintains an unofficial "best effort" internal to the GC instance called GCcode
+
+- ≈3600 projects and ≈3000 users
+- 350+ users from ESDC
+- 50 GC organizations
+- 17k+ issues and 8k merge requests
+
+(May 2019)
+
+---
+
+## Proposal
+
+To increase sharing/reuse in the GC
+
+- Evolve GCcode as an officially supported internal VCS and project management platform for the GC community, with assistance from ESDC (e.g. funding)
+- Make it the default working environment for all source code projects in ESDC
+  - Internal to GC or private projets
+
+Not just a copy of the code, it's where teams work.
+
+A stepping stone to working in the open by default.
+
+---
+
+## More than just version control
+
+- Projects (groups, repositories)
+- VCS (source code, files)
+- Issues (bugs, features, tasks)
+- Kanban (to prioritize/visualize tasks)
+- Continuous Integration (automated testing)
+- Wiki
+- Websites
+- Integration with other systems
+- ..
+
+See [Gitlab Docs](https://docs.gitlab.com/ee/README.html)
+
+---
+
+## Why not SaaS
+
+### Pros
+
+- **Leverage cloud infrastructure** - Don’t compete with providers' resources and ability to stay current
+- **Work in Complete Openness** - Allow employees to work in the open with the rest of the world
+- **Private repositories** - Private projects are available for free or as part of a subscription ($$)
+
+--
+
+### Cons
+
+- **Comfort Level with Openness** - Teams can be reluctant to work completly in the open
+- **Vendor Lock-in** - Providers's roadmap is determined based on industry funding, not by the GC's interests
+- **Potential Cost Savings** - Own instance could result in cost savings due to number of potential private repositories at GC level
+- **Contribute to GitLab CE** - Allow the GC to contribute and influence direction of the GitLab OSS project
+
+---
+
+### SaaS vs GCcode
+
+**SaaS** is available directly on the Internet and can be used to host unclassified GC information only.
+
+**GCcode** is available only on the GC network and is an instance of the open source software (OSS) Gitlab CE.
+Currently unclassified, but could be Protected B
+
+---
+
+## Benefits to ESDC – IITB
+
+- Strengthen **relationship** between IT and business through a common VCS and project management platform
+- Improve the **quality and security** of code with automated testing
+- **Accelerate** the rate of development and deployments
+- **Discover/reuse** code from other projects (incl. other depts)
+- Obtain metrics/statistics for **reporting and continuous improvement**
```

Comment:
Work in the open, internally (related to relationship between IT and business where our work is fully visible, at least internally)